File: app/src/main/java/com/tebet/mojual/view/profile/ProfileFragmentProvider.java
package com.tebet.mojual.view.profile;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ProfileFragmentProvider {

    @ContributesAndroidInjector(modules = ProfileFragmentModule.class)
    abstract ProfileFragment provideProfileFragmentFactory();
}

File: app/src/main/java/com/tebet/mojual/view/profile/ProfileFragmentProvider.kt
package com.tebet.mojual.view.profile

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ProfileFragmentProvider {

    @ContributesAndroidInjector(modules = [ProfileFragmentModule::class])
    internal abstract fun provideProfileFragmentFactory(): ProfileFragment
}
